### PR TITLE
Update workflow to not install ruby twice, remove a few unnecessary commands

### PR DESCRIPTION
**Pull request by rbclark**
This should speed up the CI time and standardize the format a little.

### CODE REVIEW

File: kitchen.ec2.yml [hunk 1]
```diff
@@ -27,54 +14,15 @@ driver:
...
-        - remote: sudo chage -d $(( $( date +%s ) / 86400 )) ec2-user
-        - remote: echo "NOTICE - updating ec2-user sudo config"
-        - remote: sudo chmod 600 /etc/sudoers && sudo sed -i'' "/ec2-user/d" /etc/sudoers && sudo chmod 400 /etc/sudoers
-
-    provisioner:
-      manifest: stig.pp
-    verifier:
-      name: inspec
-      inputs:
-        disable_slow_controls: true
```
Comment (aaronlippold) [inline kitchen.ec2.yml:80]: We need to check the `default` inputs for the profile to ensure that we are in fact disabling the long-running controls. I think we have a PR asking that this default be set back to false so in our CICD we will want to set it to `true`

File: kitchen.vagrant.yml [hunk 2]
```diff
@@ -1,71 +1,24 @@
-<%
-require 'time';
-%>
 ---
 driver:
   name: vagrant
   customize:
     natdnsproxy1: "on"
     natdnshostresolver1: "on"
```
Comment (aaronlippold) [inline kitchen.vagrant.yml:7]: These may or may not be needed now that we are off the MITRE proxy - they don't hurt but ...

File: kitchen.vagrant.yml [hunk 3]
```diff
@@ -1,71 +1,24 @@
...
-  inspec_tests:
-    - name: RedHat Enterprise Linux 7 STIG
-      path: .
-      #branch: "v2.6"
-  #controls:
-  #  - V-72309
-  input_files:
-    - inputs.example.yml
-  inputs:
-    disable_slow_controls: true
```
Comment (aaronlippold) [inline kitchen.vagrant.yml:37]: Again, we may want to review these defaults to ensure that:
a. we set sensible default with respect to the profile and STIG
b. that we create defaults that are sensible for our CICD

Comment (rbclark) [reply kitchen.vagrant.yml:37]: The `inputs.example.yml` sets `disable_slow_controls: true` as well, and the `inputs.example.yml` is still set in the default `kitchen.yml`. Will you want to change the value in `inputs.example.yml` to false as well or will that stay as is?

**Pull request merged**